PageObjectPattern/src/test/java/ShoppingStepDifinitions/ResetPasswordSteps.java

```diff
@@ -1,35 +1,66 @@
 package ShoppingStepDifinitions;
 
+import static org.junit.Assert.assertTrue;
+
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
 import io.cucumber.java.en.*;
+import shopping.HomePage;
+import shopping.LoginPage;
+import shopping.ResetPasswordPage;
 
 public class ResetPasswordSteps {
+    WebDriver driver;
+    HomePage homePage;
+    LoginPage loginPage;
+    ResetPasswordPage resetPassPage;
+
     @Given("User is on Home page")
-    public void user_is_on_home_page() {
-        System.out.println("Step 1: User is on home page");
+    public void getHomePage() {
```

Comment:
This method already define in a different class, please check

Reply:
done